config/locales/manage_assessments.en.yml

```diff
@@ -46,9 +46,11 @@ en:
         add: Add
         headline: Add assignment
     coverages:
+      coverage:
+        add_another_outcome: Add another outcome
       new:
-        add_attachment: Add an Attachment
-        add_outcome: Add an Outcome
+        add_attachment: Attach student work
```

Comment:
I think "Add an Attachment" was what was in the mockup when adding attachments to a subject (this is most likely a syllabus, not student work at this stage).